documentation/developer-guide/modules/tooling-guide/pages/bamm-cli.adoc

```diff
@@ -75,13 +81,17 @@ The available options and their meaning can also be seen in the help text of the
                                         |  _--paging-cursor-based, -pc_ : in case there is more than one paging possibility, it has to be cursor based paging|
                                         |  _--paging-offset-based, -po_ : in case there is more than one paging possibility, it has to be offset based paging|
                                         |  _--paging-time-based, -pt_ : in case there is more than one paging possibility, it has to be time based paging|
-.2+| aspect <model> to json             | Generate OpenAPI JSON specification for an Aspect Model        | `bamm aspect AspectModel.ttl to json`
+| _--custom-resolver_ : use an external resolver for the resolution of the model elements |
+.3+| aspect <model> to json             | Generate OpenAPI JSON specification for an Aspect Model        | `bamm aspect AspectModel.ttl to json`
                                         |  _--output, -o_ : output file path (default: stdout)           |
-.2+| aspect <model> to schema           | Generate JSON schema for an Aspect Model                       | `bamm aspect AspectModel.ttl to schema`
+| _--custom-resolver_ : use an external resolver for the resolution of the model elements |
+.3+| aspect <model> to schema           | Generate JSON schema for an Aspect Model                       | `bamm aspect AspectModel.ttl to schema`
                                         |  _--output, -o_ : output file path (default: stdout)           |
-.3+| aspect <model> to aas              | Generate Asset Administration Shell (AAS) submodel template for an Aspect Model  | `bamm aspect AspectModel.ttl to aas`
+| _--custom-resolver_ : use an external resolver for the resolution of the model elements |
+.4+| aspect <model> to aas              | Generate Asset Administration Shell (AAS) submodel template for an Aspect Model  | `bamm aspect AspectModel.ttl to aas`
                                         |  _--output, -o_ : output file path (default: stdout)           |
                                         |  _--format, -f_ : output file format (xml or aasx, default: xml)|
+| _--custom-resolver_ : use an external resolver for the resolution of the model elements |
 |===
 
```

Comment:
I think a separate (short) section on the custom resolver is required here. It needs to describe the basic contract when someone wants to implement their own custom resolver.